defaults/main.yml

```diff
@@ -1 +1,7 @@
 ---
+
+sansible_amazon_inspector_agent_dest: /tmp/amazon_inspector_agent
+sansible_amazon_inspector_agent_download_only: no
+sansible_amazon_inspector_agent_enabled: yes
+sansible_amazon_inspector_agent_state: started
```

Comment:
Should we just amalgamate these flags into a single start_on_boot flag?

Reply:
My thinking was that you could use the role to start / stop the agent as well, if you only want to run it when you are planning a scan.  In retrospect, I'm not sure whether that makes sense; what do you think?

Reply:
I guess it depends on how we want to use it, I was thinking we would spin up a dedicated instance from an image, apply this role and then kick off a test, once finished you'd destroy the instance; however if you want to be able to run a test on any running machine then I think these flags do indeed make sense.

Reply:
It was the the runtime behavior analysis[1] I was thinking of specifically, which monitors an instance while it's doing its thing.  Let's leave it in for the moment, I think there is value in it.

[1] https://docs.aws.amazon.com/inspector/latest/userguide/inspector_runtime-behavior-analysis.html